cli/command/service/opts.go

```diff
@@ -272,10 +222,10 @@ func (opts updateOptions) rollbackConfig(flags *pflag.FlagSet) *swarm.UpdateConf
 }
 
 type resourceOptions struct {
-	limitCPU      opts.NanoCPUs
-	limitMemBytes opts.MemBytes
-	resCPU        opts.NanoCPUs
-	resMemBytes   opts.MemBytes
+	limitCPU      dopts.NanoCPUs
```

Comment:
lol, I was actually looking at that and scratching my head if I overlooked something :smile: